meeting-notes/20181113.md

```diff
@@ -0,0 +1,203 @@
+     
+Note: Ask pedro about granularity of Discussify v1 milestones on the roadmap
+     
+ #### @andreforsousa    
+ - Concluded:
+    - Discussify:
+    - Style-guide updates with all scenarios and states (hover, focus, error feedback) for "comment-component"
+        - Comment
+        - Comment (ownership)
+        - Comment Actions
+        - Comment Edited
+        - Comment Edited History
+        - Comment Placeholder
+        - Comment Removed
+        - Comment Input (new reply)
+        - Comment Input (editing)
+    - QA & style-guide updates for the pre-alpha version
+        - FAB Shadow fix
+    - PeerPad UX/UI fixes:
+        - Grid width for 80columns accommodation (desktop)
+        - Dismissible notifications (desktop)
+        - Empty state illustration
+        - History revision
+        
+- Blocked:
+    - PeerPad mobile:
+    - Hi-Fi prototype to start looking for:
+        - Transitions between elements
+        - Start defining values for the eases
+        - This will be in the backlog until desktop version is properly defined and polished, following David’s feedback.
+    
+- In progress:
+  - Discussify Brand & Web Guidelines (manual)
+  - Look at new features for PeerPad (sync with Victor)
+
+ 
+#### @VictorBjelkholm
+ - Concluded:
+ 		- Error reporting on peerpad.net
+ - In Progress:
+ 		- UX review
+    - Pinner review
+ - Blocked:
+ - Next:
+    - Pinner deployment
+    - Metrics
+ 
+#### @aschmahmann
+ - Concluded:
+  - Roadmapping round #1
+  - Basic graph sync + updating working
+ - In progress:
+  - Looking into using existing CRDT tools like y.js to talk to the local Go graph syncing client
+     - Any advice would be appreciated
+ - Blocked:
+  - Not sure how much work to put into improving graph syncing vs wait for IPLD selectors/GraphSync
+  - A little blocked by the lack of the Wallet and Friends components of the Sharing API
+ - Next:
+  - More progress towards a causal chat application
+   - Which avenue to explore next somewhat depends on our file sharing API priorities
+    - Could also focus on writing proposals for this
+
+#### @jimpick
+ - Concluded:
+   - working group 2019 roadmap, 1st iteration (participant)
+   - fixes to ORMap CRDT
+   - Pedro's editor binding fixes
+   - real-life testing of peer-pad-nano with PL staff
+   - zombie gossip messages and hacks to filter them out
+   - React.js binding fixes (didn't work on Android!)
+   - Trying out the peer-pad e2e load tests
+   - Wildcard DNS / more PL staff testing
+ - In progress:
+   - Moving changes to production version (too diverged right now!)
+   - Pinner for production version?
+   - Product management for production version
+   - Review pinner work by Pedro
+   - Review pinner work by Dirk
+   - Characterize indexeddb perf issues
+   - Characterize zombie gossip issue
+   - Fix e2e smoke test
+   - Tokyo Nodefest next week!
+ - Blocked:
+   - Meetings - Roadmapping vs. PeerPad PM
+   - Dogfooding / testing
```

Comment:
Is PeerPad ready for other teams to dogfood it?

Reply:
I think it's pretty close. I think when we get the latest code from master deployed, and a pinner set up, we should try it out with just the DDC WG, and then test it more widely... IPFS team? Then all of PL.